observation_portal/sciapplications/viewsets.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import viewsets
+from rest_framework.permissions import AllowAny
+
+from observation_portal.sciapplications.serializers import CallSerializer
+from observation_portal.sciapplications.models import Call
+
+
+class CallViewSet(viewsets.ReadOnlyModelViewSet):
```

Comment:
The proposal details page needs call information... I'm sure I'll have to refactor this when I work on the science applications pages.